packages/commonwealth/server/models/contract_abi.ts

```diff
@@ -0,0 +1,43 @@
+import * as Sequelize from 'sequelize'; // must use "* as" to avoid scope errors
+import { DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+
+export type ContractAbiAttributes = {
+  id: number;
+  abi: string;
```

Comment:
Should this be object or string?

Reply:
is `Record<string, unknown>` valid here in your mind?